core/commands.py

```
@@ -1437,6 +1441,66 @@ def pretty(v):
+        """
+        qq_sender = getattr(self.forwarder, "qq_sender", None)
+        if qq_sender is None:
+            yield event.plain_result("QQ 发送器未初始化。")
+            return
+
+        cmd = (action or "").strip().lower()
+
+        if cmd == "on":
+            qq_sender._debug_override["__global"] = True
+            yield event.plain_result(
+                "已开启 QQ 发送诊断日志（运行时覆盖，重启后失效）。\n"
```

Ruff (0.15.12)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)